src/views/components/ChangeName.jsx

```diff
@@ -49,8 +50,14 @@ export const ChangeName = ({ showChangeNameModal, setShowChangeNameModal }) => {
   const hmsActions = useHMSActions();
   const [currentName, setCurrentName] = useState("");
   const changeName = async () => {
+    const name = currentName.trim();
+    if (name.length < 1) {
```

Comment:
is this needed with the disabled condition fix?

Reply:
This is required for the case when user presses enter with spaces in name.

Reply:
can we also add onKeyDown on input for enter key

Reply:
enter key is handled through form submit